assets/src/scss/components/_buttons.scss

```diff
@@ -261,3 +261,27 @@ button.load-more-mt {
     transition: opacity 400ms;
   }
 }
+
+.btn-post-tag {
+  --button-post-tag-- {
+    border-color: $link-color;
+    color: $link-color;
+    background-color: $white;
+    font-weight: 500;
+
+    &:visited {
```

Comment:
I think we have other places where a visited state could apply not intentionally, since [the rule](https://github.com/greenpeace/planet4-master-theme/blob/af9998a39132464a872959ffa0dbf58dcdc16906/assets/src/scss/base/_typography.scss#L116-L118) applies to every `a` element on the page.

Perhaps we can make the "main" visited selector more specific. I.e. only apply it to links in the post content?
```css
.post-content a:visited {
    color: $link-color-visited;
}

Reply:
It seems overall safer to do, since most elements outside the post content have a different background anyway, and the main link visited color is really made for against a white background.

Reply:
In this particular case it would not solve the issue, since the post tags are in the `post-content` div... But yeah maybe it would be good for other buttons! Might be better in a separate ticket since it will need some testing?

Reply:
Do the post tags need to be inside the `post-content` div? `post_content` is a specific field on a post that contains what you see in the editor.

Reply:
If we move them outside of the `post-content` div then they are no longer correctly aligned :/ Of course that could be solved by adding some duplicated CSS to them but that's not ideal

Reply:
IMHO that makes sense to do anyway. I don't think we would need to duplicate a lot (if any) CSS to do that.

Reply:
[Done](https://github.com/greenpeace/planet4-master-theme/pull/1630/commits/3e9d157f4bde38957469f9337230cd088392a678)

Reply:
However I'm not sure about using
```
.post-content a:visited {
    color: $link-color-visited;
}
```
since links could also be in blocks for example and there they would need the visited state too 🤔

Reply:
Pages also use this class, though interestingly Campaign Pages don't. `post-content` here is not referring to the "post" post type. It's called like that because the content field is called `post_content` on posts of all types.

Blocks are by definition inside of the post content, so that shouldn't be a problem.

Reply:
Right, I was put on the wrong track as I assumed this class referred to the `post_content` field which is present on posts of all types. But I see for pages it uses `page-content`. So we currently don't have a consistent class around the editor content across post types.

We could of course add one, or just target both in the CSS selector.

As for blocks, those should by definition be inside of the post content.

Reply:
Anyway, it was just a suggestion to see if it could make our live easier, seems a little more complex than I initially hoped. I'm ok with the current code and getting back to this later.